documentation/release_notes.rst

```diff
@@ -57,8 +57,9 @@ New in This Release
   with ``unseq`` or ``par_unseq`` policy when compiled by Intel® oneAPI DPC++/C++ Compiler
   with ``-fiopenmp``, ``-fiopenmp-simd``, ``-qopenmp``, ``-qopenmp-simd`` options on Linux.
   To avoid the issue, pass ``-fopenmp`` or ``-fopenmp-simd`` option instead.
-- Incorrect results may be produced by ``reduce`` and ``transform_reduce`` with 64-bit types and ``std::multiplies``,
-  ``sycl::multiplies`` operations when compiled by Intel® C++ Compiler 2021.3 and newer and executed on GPU devices. 
+- Incorrect results may be produced by ``reduce``, ``reduce_by_segment`` and ``transform_reduce``
```

Comment:
My suggestion (adding commas, version, and correct Intel(R) citation):

- Incorrect results may be produced by ``reduce``, ``reduce_by_segment``, and ``transform_reduce`` with 64-bit data types when compiled with the Intel® oneAPI DPC++/C++ Compiler versions 2021.3 and newer and executed on GPU devices.

Reply:
We do not have to put the (R) sign each time on the page, only the first time is enough, is not it? And this very trademark is already mentioned above on the page.

Reply:
We should have the (R) at the first instance on each individual page. What I would do:

- Add the (R) to the first instance on the page (first bullet)
- Remove the (R) from the sixth and seventh bullet and the new entry